threshold.js

```diff
@@ -2021,7 +2023,7 @@ const experiment = (blockCount) => {
       }
       // update/draw components on each frame
 
-      const uniDelay = 0; // 0.5 by default?
+      const uniDelay = 0;
```

Comment:
I think the `uniDelay` is for the same purpose? Maybe we can pass the `targetSafetyMarginSec` value into it? Or we can rename/delete it at all.

Reply:
No, `uniDelay` comes before the stimulus is presented. `targetSafetyMarginSec` comes after the stimulus is undrawn, before the alphabet and response-instructions show.

Reply:
Oh I see, then it's fine to leave uniDelay there